dcpy/connectors/edm/packaging.py

```diff
@@ -10,56 +7,29 @@
 from dcpy.connectors.edm import publishing
 
 BUCKET = "edm-publishing"
+BUCKET_FOLDER = "product_datasets"
 BUCKET_ACL: s3.ACL = "public-read"
+PACKAGE_FOLDER = "package"
 
-DOWNLOAD_ROOT_PATH = Path(".publishing")
+DOWNLOAD_ROOT_PATH = Path(".package")
 OUTPUT_ROOT_PATH = Path(".output")
 
 
 @dataclass
 class PackageKey(publishing.ProductKey):
     product: str
     version: str
+    dataset: str
```

Comment:
Isn't this more a PackageDatasetKey then? Just that PackageKey sounds like it is a unique key for a package, not a dataset within it

Reply:
That's a good point. We really have two types of Package: The `ProductPackage` and the `DatasetPackage`. Will change that to make it clear here.